resources/shaders/shadow_ray.hlsl

```diff
@@ -10,7 +10,7 @@ struct Attributes { };
 
 bool TraceShadowRay(uint idx, float3 origin, float3 direction, float far, unsigned int depth)
 {
-#ifndef NO_SHADOWS
+//#ifndef NO_SHADOWS
```

Comment:
If you can already disable/enable shadows through other methods, this has to be removed. Otherwise this should be left here and not commented.